### DISCUSSION TITLE
I have a question for AutoWireViewModel

### DISCUSSION THREAD

**Discussion by L261173157** [PrismLibrary/Prism#2252] [WPF]
when I debug my app.
I got :

System.Windows.Markup.XamlParseException:““设置属性“Prism.Mvvm.ViewModelLocator.AutoWireViewModel”时引发了异常。”，行号为“7”，行位置为“9”。”

what can I do for it ???

**Comment by LGTCO** [comment]
ViewModel need to inherit from 'BindableBase' 
？

**Reply by L261173157** [reply]
I have found my mistake.my viewmodel is a constructor with parameter.
parameter is IEventAggregator.But it is't by registryed.
and I don't know how to registry with IOC.
I'm learning on https://prismlibrary.com/docs/event-aggregator.html
But I didn't get it.

**Reply by LGTCO** [reply]
![image](https://user-images.githubusercontent.com/23730062/99218255-f8967900-2814-11eb-8131-91fe9695b864.png)
you can find IEventAggregator and EventAggregator here

**Reply by L261173157** [reply]
I great a new IEventAggregator.cs. Finally I found that I can using Prism.Events to do within sample codes.
Thank you ,I'm ok.
